pkg/trace/config/config.go

```diff
@@ -464,7 +464,7 @@ func New() *AgentConfig {
 
 		ReceiverHost:           "localhost",
 		ReceiverPort:           8126,
-		MaxRequestBytes:        50 * 1024 * 1024, // 50MB
+		MaxRequestBytes:        25 * 1024 * 1024, // 25MB
```

Comment:
Nitpick: We can mention in the comment that `25MB` is the limit enforced by the APM backend.

Reply:
So the backend limit isn't 25MB (it is intentionally not quite the same and I don't know if we want to make public any exact limit as we could possibly change the limit in the future?)

Reply:
Fair point! The comment is for the DD devs who'll read the code in the future, to help them understand the magic number `25MB`, not for users.